client/src/components/AppHeader/AppHeader.js

```diff
@@ -1,49 +1,107 @@
 import React from 'react';
 
 import '../../styles/index.scss';
+import logo from '../../assets/images/logo.png';
 
-import { Link } from "react-router-dom";
-import { Navbar,  FormControl, Form, Nav, InputGroup } from 'react-bootstrap';
-import { faSignInAlt, faUsers, faCalendarAlt } from '@fortawesome/free-solid-svg-icons';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { AppBar, Toolbar, IconButton, InputBase, Link } from '@material-ui/core';
+//You can find icon names here: https://jxnblk.com/rmdi/
+import { EventAvailable, SupervisorAccount, PersonAdd, Person, MoreVert, Search } from '@material-ui/icons';
+import { Link as RouterLink } from 'react-router-dom';
 
-import logo from '../../assets/images/logo.png';
+class AppHeader extends React.Component {
+    state = {
+        mobileMenuOpened: false,
+    };
+
+    handleMobileMenuToggle = event => {
+        this.mobileMenuOpened = !this.mobileMenuOpened;
+        this.setState({ mobileMenuOpened: this.mobileMenuOpened });
+    };
 
-const AppHeader = props => (
-        <Navbar collapseOnSelect expand="lg" className = 'header'>
-            <Navbar.Brand href="/" className="mobile-logo d-block d-lg-none">
-                <img className = 'logoImg' src={logo} alt="Logo" />
-            </Navbar.Brand>
-            <Navbar.Toggle aria-controls="responsive-navbar-nav" className="toggle" />
-            <Navbar.Collapse id="responsive-navbar-nav">
-                <Nav inline className="header-content">
-                    <div className="search-row">
-                        <Navbar.Brand href="/" className="site-logo d-none d-lg-block">
-                            <img className = 'logoImg' src={logo} alt="Logo" />
-                        </Navbar.Brand>
-                        <Form inline className="search-form">
-                            <InputGroup>
-                                <FormControl className="search-field" placeholder="Search"/>
-                            </InputGroup>
-                        </Form>
-                    </div>
-                     <div className="align-center">
-                        <Link to="/events" className="fa-icon">
-                            <FontAwesomeIcon icon={faCalendarAlt}/>
-                        </Link>
+    render() {
+        const { mobileMenuOpened } = this.state;
+        const renderMobileMenu = (
+            <div className={"mobile-menu" + (mobileMenuOpened ? "" : " hidden")}>
```

Comment:
Optional, <div className={"mobile-menu" && mobileMenuOpened + " hidden")}> -  shorter.

Reply:
If you do not mind, I will leave the original version because, I think that, ?: looks better than "mobile-menu" + (!mobileMenuOpened && " hidden" || "")"

Reply:
ok